opm/simulators/linalg/bda/WellContributions.cpp

```diff
@@ -17,15 +17,14 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-
 #include <config.h> // CMake
 #include <cstdlib>
 #include <cstring>
-
+#include <iostream>
```

Comment:
This is probably a remnant of debugging (with std::cout/std::cerr). Please remove